app/models.py

```diff
@@ -2227,10 +2227,10 @@ def unreject_approval(self):
 
         self.status = 'submitted'
 
-    def create_approval_task(self):
+    def create_approval_task(self, closing_date=None):
         if current_app.config['JIRA_FEATURES']:
-            mj = get_marketplace_jira()
-            mj.create_application_approval_task(self)
+            mj = get_marketplace_jira(closing_date is None)
```

Comment:
False means user/password authentication test mode here, you need to use () (no parameters) in production